packages/writing-style/README.md

```diff
@@ -0,0 +1,36 @@
+# commercetools Writing Style Rules
+
+This package provides
+
+- commercetools specific writing style and terminology rules for the configurable [vale](https://errata-ai.github.io/vale/) prose linter.
+- the `commercetools-vale` command line tools which wraps a bundled vale build and calls it using the included writing style and configuration
+
+The base style is the [Google Developer Documentation Style Guide](https://github.com/errata-ai/Google), which is included as a copy (MIT licensed, too) and modified and enriched with commercetools specific rules.
+
+The configuration defaults to checking `.md`, `.mdx`, `.txt`, and `.html` files, which can be overridden by calling it with a command-line parameter `commercetools-vale --glob='*.{js,md}' ./path/to/content` passing a list of [supported file format extensions](https://errata-ai.github.io/vale/formats/#formats).
+
+> As of Nov 2019, basing off Google's styles was a pragmatic choice. Other styles like Microsoft's are attractive starting points, too.
+
+## Linter Usage: Standalone and via Editor Plugins
+
+1.  Install the command globally via `npm install -g @commercetools-docs/writing-style`
+1.  Use it by calling `commercetools-vale ./path/to/your/content`
+
+In addition, you can install a matching editor plugin to get immediate feedback while authoring, for example [for VSCode](https://marketplace.visualstudio.com/items?itemName=testthedocs.vale) - don't use the `vale-server` one but the one for the command line.
+
+**If your plugin allows, reconfigure the command from `vale` to `commercetools-vale`**. Otherwise, for example in VSCode, you have to uninstall your global vale installation and create an alias via `echo "alias vale=commercetools-vale" >>~/.bash_profile` (or ending `.zsh` if you are on a fresh Mac OS X catalina) (no insurance provided, suggestions for a cleaner method welcome).
+
+Please don't forget to frequently update the package to get the latest styles.
+
+## Linter Usage: CI and command line in Javascript Projects
```

Comment:
I see 2 problems at the moment:
- the vale binaries that we ship with the package are not versioned, meaning that it's not clear which version of vale is running under the hood
- the vscode plugin does not really work at the moment as it assumes the binary is installed on the global `PATH`. So if we install the package in a repository, we also need to install it globally in order to use the editor plugin.
  I can look into the vscode plugin and open a PR to make the PATH configurable, so that we can point it to the workspace installation.

Reply:
This is what we should aim for in my opinion (regarding vscode): https://github.com/microsoft/vscode/blob/2aa183fdc89d93bbb592183925ec3c797fc46ac7/.vscode/settings.json#L40

Reply:
concerning 1:   I can rename them to a versioned file name if that makes it more transparent. In any case you can just call  it with `--version` and get the version.  Not sure about the relevance though. 

concerning 2: Yes, I suspected that, too.  The plugin uses execFile to call vale, so the setting the shell environment does not work because execFile does not spawn a shell to call the binary. 
The call is hardwired here: https://github.com/testthedocs/vscode-vale/blob/master/src/extension.ts#L156   Ideal would be to be able to configure a different executable path per workspace (not even going through the PATH env).  I've never done VSCode plugins, would have to learn how to declare a configuration flag.

Reply:
PS: I created an issue with a proposal:  https://github.com/testthedocs/vscode-vale/issues/7

Reply:
Thanks, I can also spin up a PR on their repo today. It should be pretty straightforward to add the setting.

Reply:
I addressed both issues now.  The README does not promise any more that the VSCode plugin already works and the binaries have the version number in the file name now.